jogo42-cb/jogador.c

```diff
@@ -47,6 +47,12 @@ static bool ColisaoJogLevel(const Vector2 posJogTeste, const GameState* gs)
         }
     }
 
+    //[ CHECAR CONTRA O INIMIGO ]-----------------------------------------------
+    if (CheckCollisionCircles((Vector2){gs->inim.pos.x, gs->inim.pos.y}, 20, posJogTeste, RAIO_JOG))
```

Comment:
`gs->inim.pos` já é um `Vector2`, então `(Vector2){gs->inim.pos.x, gs->inim.pos.y}` tem valor igual e é redundante.

```suggestion
    if (CheckCollisionCircles(gs->inim.pos, 20, posJogTeste, RAIO_JOG))
```